SCSH_3C_iOS/SCSH_3C_iOS/Repository/Model/Theme.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Theme.swift
+//  SCSH_3C_iOS
+//
+//  Created by 辜敬閎 on 2023/10/25.
+//
+
+import Foundation
+
+protocol ThemeType {}
+
+enum HomeTheme: ThemeType {
+    case Explore
+    case Subject
+    case _3C
```

Comment:
Does this qualify as a violation of naming conventions? 🤣 

How about using `Product3C` instead?

Reply:
Agree !! I don't know why I do that 🤣

> How about using Product3C instead?

ok !

Reply:
[style: renaming HomeTheme](https://github.com/No-57/SCSH_3C_iOS/pull/32/commits/233b01545e2d98edee698ae18843490b31970a95)